### PR TITLE
Improve cmdline tool docs

### PR DESCRIPTION
**Pull request by zachlewis**
Addresses #1115 

Provides more elaborate descriptions of sparsely documented OCIO command-line tools.
(ociolutimage, ocioconvert, ociodisplay, and ocioperf).

(work in progress!)

### CODE REVIEW

File: docs/guides/using_ocio/tool_overview.rst [hunk 1]
```diff
@@ -160,24 +176,44 @@ supported.
 ociodisplay
 ***********
 
-A basic image viewer. Uses OpenImageIO to load images, and displays them using
-OCIO and typical viewer controls (scene-linear exposure control and a
-post-display gamma control)
+A toy image viewer demonstrating the OCIO C++ API.
```
Comment (michdolan) [inline docs/guides/using_ocio/tool_overview.rst:179]: Suggest changing `toy` to `example` or `simple` for clarity.

File: docs/guides/using_ocio/tool_overview.rst [hunk 2]
```diff
@@ -160,24 +176,44 @@ supported.
...
+A toy image viewer demonstrating the OCIO C++ API. 
 
-May be useful to users to quickly check colorspace configuration, but
-primarily a demonstration of the OCIO API
+Uses OpenImageIO to load images, and displays them using OCIO and 
+typical viewer controls (scene-linear exposure control and a
+post-display gamma control).
 
-.. TODO: Link to more elaborate description
+May be useful to users to quickly check colorspace configuration.
```
Comment (michdolan) [inline docs/guides/using_ocio/tool_overview.rst:185]:
```suggestion
May be useful to users to quickly check a color space configuration.
```

File: docs/guides/using_ocio/tool_overview.rst [hunk 3]
```diff
@@ -211,7 +247,28 @@ The ocioperf tool allows you to benchmark the performance of a given color
 transformation on your hardware.  Please use the --help argument for a 
 description of the options.
 
-.. TODO: Link to more elaborate description
+The metric used for assessing performance is the time taken to apply a 
+transform to an image with respect to each pixel, to each line, or to the 
+entire image plane (or all three). By default, each test is taken ten times.
```
Comment (michdolan) [inline docs/guides/using_ocio/tool_overview.rst:252]:
```suggestion
entire image plane (or all three). By default, each test is run ten times. 
```

**Comment by michdolan** [comment]
@zachlewis this is great! I think this is headed in the right direction and very helpful for understanding the tools beyond the `--help` content. Thanks for your work on this. I suggested some possible wording tweaks, but otherwise LGTM.

**Comment by zachlewis** [comment]
Thanks Michael! I appreciate you taking the time to look at this. I'll make those changes right now.

File: docs/guides/using_ocio/tool_overview.rst (hunk 1 above)
Comment (zachlewis) [reply docs/guides/using_ocio/tool_overview.rst]: Agreed. "Toy" is pretty colloquial.
I wanted to make explicit that `ociodisplay` isn't (currently?) intended as full-fledged demo of all of OCIO's viewing related features (for instance, presently, it's not a reference for implementing UIs for dynamic transforms); and I'm not sure whether there are plans to update `ociodisplay` further.

If `ociodisplay` intends to serve as reference of developers, I feel like it deserves separate developer-oriented documentation (see #447), and possibly an "* ^kitchen-sink" reference config, to help DCC devs better understand how to support OCIO in practice.

**Review by doug-walker — APPROVED** [review]

**Comment by hodoulp** [comment]
@zachlewis Based on the positive feedback It seems that your work can now be a pull request (i.e. not a draft). I suggest that you change the status so we can merge it when approved. @michdolan could you finalize your review?

**Review by michdolan — APPROVED** [review]

**Pull request merged**